docs/user_guides/mlops/registry/model_evaluation_images.md

```diff
@@ -0,0 +1,79 @@
+---
+description: Documentation on how to attach model evaluation images to a model.
+---
+
+# How To Save Model Evaluation Images
+
+## Introduction
+
+In this guide, you will learn how to attach ==model evaluation images== to a model. Model evaluation images contain **confusion matrices**, **ROC curves** or other graphs that help visualizing model evaluation metrics. By attaching model evaluation images to your model version, other users can better understand the experiment results obtained during model training.
```

Comment:
Model evaluation images are images that visually  describe model performance metrics. For example, **confusion matrices**, **ROC curves**, **model bias tests**, and **training loss curves** are examples of common model evaluation images.

By attaching model evaluation images to your versioned model